wafer/pages/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from reversion import revisions
+
+from wafer.pages.models import Page
+
+
+class PageSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Page
+        exclude = ('_content_rendered',)
+
+    @revisions.create_revision()
+    def create(self, validated_data):
+        revisions.set_comment("Created via REST api")
+        return super(PageSerializer, self).create(validated_data)
+
+    @revisions.create_revision()
+    def update(self, page, validated_data):
+        revisions.set_comment("Changed via REST api")
+        page.parent = validated_data['parent']
+        page.content = validated_data['content']
```

Comment:
Do we want to allow updating of other attributes like `include_in_menu`, `exclude_from_static` and `people`? Maybe `files` too but perhaps that should be left for a later pull request.